GVFS/GVFS.Common/ProductUpgrader.cs

```diff
@@ -261,14 +262,10 @@ public virtual bool TryLoadRingConfig(out string error)
                 }
                 else
                 {
-                    error = "Invalid upgrade ring `" + ringConfig + "` specified in Git config.";                    
+                    error = "Invalid upgrade ring `" + ringConfig + "` specified in Git config.";

```

Comment:
It's no longer in the "Git config". Please scrub any other messages that say this.